config/manager/kustomization.yaml

```diff
@@ -22,6 +22,8 @@ images:
   newName: quay.io/kiegroup/kogito-serverless-operator-nightly
   newTag: latest
 # Patching the manager deployment file to add an env var with the operator namespace in
+#        - name: ENABLE_WEBHOOKS
+#          value: "true"
```

Comment:
Why is this commented? Shouldn't this be a part of the patch?

Reply:
It should, will remove it.